xbstrap/schema.yml

```diff
@@ -161,6 +161,8 @@ properties:
                 type: string
             'everything_by_default':
                 type: boolean
+            'mandate_hashes':
+                type: boolean
```

Comment:
I think we should add `_for_archives` or something similar here since we do not mandate hashes for VCS repositories.

Reply:
I'd like to allow hashing of non-rolling git trees at some point

Reply:
Yes, but even then it would not be mandated by the same option (to avoid a backwards incompatible change).

Reply:
ah, fair enough